src/software/world/game_state.h

```diff
@@ -51,7 +51,7 @@ class GameState
     // Robocup SSL Rules 9.2.
     Point ball_placement_point;
 
-    GameState() : state(HALT), restart_reason(NONE), our_restart(false) {}
+    GameState() : state(HALT), restart_reason(NONE), game_state(RefboxGameState::HALT), our_restart(false) {}
```

Comment:
I'd tried this fix, but it wasn't sufficient

Reply:
This was all I intended to change, looking into it now........